flycheck.el

```diff
@@ -9726,11 +9726,16 @@ See URL `http://www.scalastyle.org'."
                        ((not config-file) '(bold error))
                        (t 'success)))))))
 
+(flycheck-def-args-var flycheck-scheme-chicken-args scheme-chicken
+  :package-version '(scheme-chicken . "0.31"))
```

Comment:
`:package-version` records the *Flycheck* version this variable has been added.  So it should rather be `'(flycheck . "32")`.

Reply:
Changed to `'(flycheck . "32")`.